ex2.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def vector_with_evenly_distributed_values_from_5_50():
+    """
+    Function creates vector with evenly_distributed values between 5 - 50
+    using numpy lin-space function.
+    :return: Evenly_distributed vector.
+    """
+    vector = np.linspace(10, 49, 5)
```

Comment:
I think you might have misread the assignment. You were asked to create a  a vector of length 10 with values evenly distributed between 5 and 50. You created a vector of length 5 wiht values evenly distributed between 10 and 49.